cmd/cache.go

```diff
@@ -45,11 +45,8 @@ var (
 func initCache() error {
 	err := config.InitServer()
 	cobra.CheckErr(err)
-	err = metrics.SetComponentHealthStatus("xrootd", "critical", "xrootd has not been started")
-	cobra.CheckErr(err)
-	err = metrics.SetComponentHealthStatus("cmsd", "critical", "cmsd has not been started")
-	cobra.CheckErr(err)
-
+	metrics.SetComponentHealthStatus(metrics.OriginCache_XRootD, metrics.StatusCritical, "xrootd has not been started")
```

Comment:
Do we lose anything here by not checking the error from this? The previous `cobra.CheckErr` performs an `os.Exit(1)` if an error exists, so if we don't redefine `err` at the call to `metrics.SetComponentHealthStatus()`, this function will always return nil.

Reply:
I think that's the expected behavior. Now with the enum type ensured, we don't return anything from `SetComponentHealthStatus`. The error returned before is simply checking if our status string literals matches what we expect.